packages/xrpl/src/models/common/index.ts

```diff
@@ -119,7 +119,7 @@ export interface NFTOffer {
 }
 
 /**
- * One NFToken that might be returned from either an {@link NFTInfoResponse}
+ * One NFToken that might be returned from a {@link NFTInfoResponse}
```

Comment:
nit: (I think "N" is pronounced as "en" so has an implicit vowel in front so uses "an" in this case)
```suggestion
 * One NFToken that might be returned from an {@link NFTInfoResponse}
```